source/trunk/Esf.sln

```diff
@@ -0,0 +1,65 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 14
```

Comment:
I thought you would be integrating the concept project directly into Esf.Web. Is this a separate step we'll do later or this is just a throw-away prototype?